Audacia.Quest/Audacia.Quest.Core/Events/IEvent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
```

Comment:
You may want to run through all .cs files and do a tree shake (remove any unused using statements).
The web assembly compiler didn't used to do tree shaking, but I think it does now for .net 6.
I would recommend you still do it in general as it removes any extra bulk from your code.